Libraries/ActionSheetIOS/RCTActionSheetManager.m

```diff
@@ -149,33 +149,28 @@ - (void)presentViewController:(UIViewController *)alertController
     item.target = self;
     [menu addItem:item];
   }
-  
-  NSPoint origin = NSZeroPoint;
-  NSEvent *event = nil;
+
   RCTPlatformView *view = nil;
   if (anchorViewTag) {
     view = [self.bridge.uiManager viewForReactTag:anchorViewTag];
-    event = [view.window currentEvent];
   }
-  NSView *superview = [view superview];
-  if (event && view) {
-    // On a macOS trackpad, soft taps are received as sysDefined event types. SysDefined event locations are relative to the screen so we have to convert those separately.
-    NSPoint eventLocationRelativeToWindow = NSZeroPoint;
-    CGPoint eventLocationInWindow = [event locationInWindow];
-    if ([event type] == NSEventTypeSystemDefined) { // light tap event relative to screen
-      eventLocationRelativeToWindow = [[view window] convertRectFromScreen:NSMakeRect(eventLocationInWindow.x, eventLocationInWindow.y, 0, 0)].origin;
-    } else { // full click events are relative to the window
-      eventLocationRelativeToWindow = eventLocationInWindow;
+  CGPoint location = CGPointZero;
+  NSView *inView = nil;
+  if (view != nil) {
+    // Display under the anchorview
+    inView = view;
+    CGRect bounds = [view bounds];
+
+    if ([view userInterfaceLayoutDirection] == NSUserInterfaceLayoutDirectionRightToLeft) {
+      location = CGPointMake(NSMaxX(bounds), NSMaxY(bounds));
+    } else {
+      location = CGPointMake(NSMinX(bounds), NSMaxY(bounds));
     }
```

Comment:
the most minor of improvements: CGPointMake(.[x], NSMaxY(bounds)) is duplicated. Could further simplify with
```
NSPoint originX = [view userInterfaceLayoutDirection] == NSUserInterfaceLayoutDirectionRightToLeft ? NSMaxX(bounds) : NSMinX(bounds);
location = NSMakePoint (originX, NSMaxY(bounds));
```